fastlane/Snapfile

```diff
@@ -0,0 +1,50 @@
+# Uncomment the lines below you want to change by removing the # in the beginning
+
+# A list of devices you want to take the screenshots from
+# devices([
+#   "iPhone 8",
+#   "iPhone 8 Plus",
+#   "iPhone SE",
+#   "iPhone X",
+#   "iPad Pro (12.9-inch)",
+#   "iPad Pro (9.7-inch)",
+#   "Apple TV 1080p"
+# ])
+
+# XXX do we want more devices?
```

Comment:
for sure we want iPad. I don't think iPhone 8 is useful. Maybe iPhone 8 Plus could be